src/linear_algebra/basic.lean

```diff
@@ -1946,7 +1946,7 @@ protected def uncurry :
   (V → V₂ → R) ≃ₗ[R] (V × V₂ → R) :=
 { map_add'  := λ _ _, by { ext ⟨⟩, refl },
   map_smul' := λ _ _, by { ext ⟨⟩, refl },
-  .. equiv.arrow_arrow_equiv_prod_arrow _ _ _}
+  .. equiv.curry _ _ _}
```

Comment:
Shouldn't this definition be called `linear_equiv.curry` rather than `linear_equiv.uncurry`, given it builds on `equiv.curry`?

Reply:
```suggestion
  .. (equiv.curry _ _ _).symm }
```

Reply:
I'd be tempted to flip the direction of this one to match though instead.